### ISSUE TITLE
Terminal tabs are shown as separate windows in AltTab

### ISSUE THREAD

**Issue by KhaledElTahan** [lwouis/alt-tab-macos#2136]
**Describe the bug**

Terminal tabs are shown as separate windows in AltTab.

**Screenshots / video**

https://user-images.githubusercontent.com/21336749/202537741-62e673b2-57db-4f58-9ede-715af6ed5f3a.mov

![image](https://user-images.githubusercontent.com/21336749/202537037-574d12ff-0262-4909-944b-bb52fdc0b18b.png)

**Steps to reproduce the bug**

1. Open Terminal
2. Open different tabs
3. Click on the tabs with the mouse pointer
4. Use the alt tab
5. You will see that each tab is shown as a different window

**Your environment**

* AltTab version: 6.51.0
* macOS version: 12.6

**Comment by lwouis** [comment]
I'm not able to reproduce the issue.

**Comment by KhaledElTahan** [comment]
Try to click on the tabs inside the terminal with the mouse pointer, then check again?

**Comment by lwouis** [comment]
I tried, and it just works. Not sure what's different on your machine

**Comment by lawand** [comment]
Can confirm that this happens with Finder tabs. I'm on AltTab version: 6.51.0 / macOS version: 12.6

**Comment by KhaledElTahan** [comment]
The only thing that worked for me, is reverting to AltTab version: 6.49.0, as the problem started showing from 6.50.0 for me.

**Comment by probablykasper** [comment]
It happens for me with Finder windows too, but only after some time - after starting/restarting AltTab it doesn't immediately happen.

**Comment by tarunkant** [comment]
Yes, I have also the same issue(command tabs are shown as separate windows) in my M1 Max and using AltTab 6.50.0 and 6.51.0.
For fixing this issue I have downgraded the AltTab to the older version 6.49.0.

Please fix this issue as soon as possible because this is too annoying.

**Comment by khour** [comment]
> It happens for me with Finder windows too, but only after some time - after starting/restarting AltTab it doesn't immediately happen.

It happens after you do some manipulations in Finder. For instance, close AltTab, open Finder with 2 tabs, open AltTab -> one thumbnail. But if you change between Finder tabs, or add a 3rd one - then there are 2 (or 3 in the second example) thumbnails in AltTab switcher.

**Comment by GHopperMSK** [comment]
I noticed the same problem with `textEdit`
Open textEdit, add a few tabs. Use shortcut to see AltTab and will see not one but many textEdit thumbnails.
Confirm Finder and Terminal as well!

**Comment by victorlin** [comment]
I saw this occasionally up through version 6.46.0, and a restart of AltTab usually fixed it. After updating to 6.51.0, now it is happening even after restart.

**Comment by peterpp** [comment]
The same here. AltTab 6.51.0, macOS 12.6.1. Tabs in Terminal and Finder are shown as windows.

**Comment by chavlji** [comment]
Same here. Terminal, Finder, Fork. I have option `Show standard tabs as windows` off. It happened after latest update.
v6.51.0
OS Big Sur 11.5.2, MB16"2019

**Comment by zzJinux** [comment]
Same here. Terminal, Finder. For every tabs opened after AltTab launched.

AltTab v6.51.0
Monterey 12.6.2, MBP16″2021

**Comment by lwouis** [comment]
Closing as duplicate of https://github.com/lwouis/alt-tab-macos/issues/2017
